components/email_aliases/email_aliases.mojom

```diff
@@ -29,6 +29,12 @@ struct Alias {
   array<string>? domains;
 };
 
+// Return value for EmailAliasesService::GenerateAlias()
+union GenerateAliasResult {
+  string? alias_email;
+  string? error_message;
```

Comment:
nit: I don't think these should be optional


```suggestion
  string alias_email;
  string error_message;
```

Reply:
you are right -- fixed